Appraisals

```diff
@@ -0,0 +1,16 @@
+appraise '5.2' do
+  gem 'rails', '~> 5.2'
+  gem 'sqlite3', '~> 1.3.6'
+end
+
+appraise '6.0' do
+ gem 'rails', '~> 6.0'
+end
+
+appraise '6.1' do
+ gem 'rails', '~> 6.1'
+end
+
+appraise '7' do
+  gem 'rails', '7.0.0alpha2'
```

Comment:
@EugZol once Rails 7 will be official released, I'll do a PR here to update the version number